flexeval/core/language_model/base.py

```diff
@@ -22,7 +22,11 @@ def batch_complete_text(
         Args:
             text_list: A list of input texts.
             stop_sequences: A string or a list of strings that will stop the generation when they are generated.
+                This argument exists to give a common interface to various models that have different names for it.
             max_new_tokens: The maximum number of tokens to generate for each text.
+                This argument exists to give a common interface to various models that have different names for it.
+            **kwargs: Additional keyword arguments for the generation.
+                If the default generation kwargs set to the model, this will override them.
```

Comment:
```suggestion
                 This dictionary will override any existing keys in `default_gen_kwargs` of LanguageModel.
```

全てoverrideされるのではなく、unionになることを明示したく書き直してみました。

Reply:
> Note that when the same parameters are specified in gen_kwargs from EvalSetup, default_gen_kwargs will be overridden.

自分の提案よりも、むしろこれがdocstringに書いてあればいい気がします。

Reply:
もうちょっと明示した方が良かったですね。
`stop_sequences` 関係だけは重複してもマージされるということで、その点も含むように書き直しました。

https://github.com/sbintuitions/flexeval/pull/63/commits/31c59cc34735778802b7a69a65bfa059c4ac0a88

Reply:
なるほどです。めちゃめちゃいい感じだと思います。